app/models/job_proxy_dispatcher.rb

```diff
@@ -90,6 +91,36 @@ def dispatch
     _log.info "Complete - Timings: #{t.inspect}"
   end
 
+  def dispatch_container_jobs
+    jobs_to_dispatch, = Benchmark.realtime_block(:pending_container_jobs) { pending_jobs(ContainerImage) }
+    Benchmark.current_realtime[:container_jobs_to_dispatch_count] = jobs_to_dispatch.length
+
+    concurrent_container_scans_limit = Settings[:container_scanning][:concurrent_per_ems].to_i
+    if jobs_to_dispatch.length > 0
+      jobs_to_dispatch.each do |job|
+        if concurrent_container_scans_limit > 0 &&
+           active_scans(ContainerImage)[@zone] >= concurrent_container_scans_limit
```

Comment:
@jrafanie I think the existing code for vms allows -1 === unlimited ?
in that case the first conditional would prevent the break.

As I understand [existing](https://github.com/ManageIQ/manageiq/blob/master/app/models/job_proxy_dispatcher.rb#L38) code for vms, -1 OR 0 means unlimited? Is that something I should aim to imitate? 
I think I would expect concurrent_something = 0 to mean never do something.